generator/generator_tests/common.cpp

```diff
@@ -1,4 +1,7 @@
 #include "generator/generator_tests/common.hpp"
+#include <generator/osm2type.hpp>
```

Comment:
""

also, empty line after the first line, please

Reply:
++

Reply:
s/</"/
s/>/"/

Reply:
++